Lucerne/MsEth1.xml

```diff
@@ -48,33 +49,48 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                            <material key="parchment"/>
                         </support>
                         <extent>
-                           <measure unit="leaf">12</measure>
-                           <note>dimensions for the folded book</note>
+                           <measure unit="leaf">3</measure>
+                           <measure unit="page">26</measure>
+                           <note>3 leaves, bound and folded to 27 pages, the last page being glued and bound to the back board and not counted in this
+                              description. Dimensions refer to the folded book.</note>
                            <dimensions type="outer" unit="mm">
                               <height>52</height>
                               <width>42</width>
-                              
                            </dimensions>
                         </extent>
-                        <foliation></foliation>
-                        <condition key="good"></condition>                        
+                        <foliation>No foliation</foliation>
+                        <collation>Three slender pieces of parchment have been tied together and folded into 27 pages, the first and the last  of which
+                           are attached to the cover and left blank.</collation>
+                        <condition key="good"/>                        
                      </supportDesc>
-                     
-                  </objectDesc>                  
-                  <handDesc><handNote xml:id="h1" script="Ethiopic"></handNote></handDesc>
-                  
-                  
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="10"/>                            
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>                  
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">Script of the 19th or 20th century</seg>
+                        <seg type="ink">Black, red</seg>
+                        <seg type="rubrication">Incipits, name of the owner</seg>
+                        <note>Long and slender script.</note>
```

Comment:
It is better to transfer the content of these notes into a single `<desc>`

Reply:
Done.